.env.template

```
@@ -0,0 +1,3 @@
+#API_URL=http://127.17.0.1:8000/
+#API_CLIENT_ID=""
+#API_CLIENT_SECRET=""
```

```suggestion
API_URL=http://127.17.0.1:8000/
API_CLIENT_ID=""
API_CLIENT_SECRET=""
``` 

Nit, maybe we uncomment these so there's one fewer step to edit the values once the user copies `.env.template` to `.env`?

---

Eh I'm not sold on this, and it's not important.